package.json

```diff
@@ -0,0 +1,47 @@
+{
+	"name": "bookbrainz-import",
+	"version": "1.0.0",
+	"description": "Import scripts for various data sources ",
+	"main": "index.js",
+	"scripts": {
+		"build": "rimraf lib/* && babel src --out-dir lib",
+		"nodemon": "nodemon --watch './src/' --exec 'npm run build'",
+		"lint": "eslint ./src"
+	},
+	"repository": {
+		"type": "git",
+		"url": "yes "
```

Comment:
Does this need an actual URL?

Reply:
Ah, yes. This is the default value. I'll change it.